src/components/Profile/MyPosts/MyPostsContainer.jsx

```diff
@@ -1,33 +1,32 @@
 import React from "react";
 
-import ACTION_TYPES from "../../../redux/actionTypes";
-
 import MyPosts from "./MyPosts";
+import { connect } from "react-redux";
+import {
+  addPostCreator,
+  updateNewPostTextCreator,
+} from "../../../redux/actionCreators";
 
-const MyPostsContainer = (props) => {
-  console.log("mypostcontainer", props);
-  const { state, dispatch } = props;
-  const { profilePage } = state;
-  const { postsData, newPostText } = profilePage;
+let mapStateToProps = (state) => {
```

Comment:
Consider using "const" instead of "let" here.

Reply:
done